src/less-style/tree-content.less

```diff
@@ -89,10 +89,6 @@
   }
 }
 
-#jstree-marker {
```

Comment:
Found where this was added (https://github.com/dimagi/Vellum/pull/291), looks like it was needed to override full-screen mode's z-index. Full-screen no longer uses z-index, so this should be safe to remove. Tested that I can drag & drop questions in full-screen mode.